File: frontend/model/formacaoAcademica.js
class FormacaoAcademica{
    constructor(tipoFormacao, curso, instituicao, dataIngresso, dataConclusao){
        this._id;
        this._tipoFormacao = tipoFormacao;
        this._curso = curso;
        this._instituicao = instituicao;
        this._dataIngresso = dataIngresso;
        this._dataConclusao = dataConclusao;
    }

    get id(){
        return this._id;
    }

    get tipoFormacao(){
        return this._tipoFormacao;
    }

}

File: backend/src/main/java/com/g5tech/api/model/FormacaoAcademica.java
package com.g5tech.api.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "tb_formacao_academica")
@Getter
@Setter
public class FormacaoAcademica {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    @ManyToOne
    @JoinColumn(name = "id_candidato")
    private Candidato candidato;

    @Column(name = "tipo_formacao")
    private String tipoFormacao;

    @Column(name = "curso")
    private String curso;

    @Column(name = "instituicao")
    private String instituicao;

    @Column(name = "data_ingresso")
    private Date dataIngresso;

    @Column(name = "data_conclusao")
    private Date dataConclusao;
}
